gensim/models/keyedvectors.py

```diff
@@ -752,8 +752,7 @@ def wmdistance(self, document1, document2):
             If `pyemd <https://pypi.org/project/pyemd/>`_  isn't installed.
 
         """
-        if not PYEMD_EXT:
-            raise ImportError("Please install pyemd Python package to compute WMD.")
+        from pyemd import emd
```

Comment:
How does this affect existing code that relied on `PYEMD_EXT`? Do we still even need `PYEMD_EXT`, where is it used?

Reply:
we can remote `PYEMD_EXT` from wherever it is used in the project with proper imports. Is that what you mean here?

Reply:
Yes. As I remember, this also used in tests, but this isn't imported from here, i.e. filled directly in test (I don't think that we need the same change in the tests, only here).

Reply:
Ok sir, so please let me know what else to do. I have removed the import from the start now.